main.py

```diff
@@ -1,16 +1,11 @@
-# This is a sample Python script.
+# main.py
+from functions.methods import read_the_file
 
-# Press Shift+F10 to execute it or replace it with your code.
-# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
 
+def main():
+    file_path = "resources/grades1.txt"
+    read_the_file(file_path)
```

Comment:
Let's work further on single responsibility of functions.

- `read_the_file` should only read the file are return its content in the collection chosen by you
- then, there should be a function that parses data read, calculates averages and return them in one structure
- and finally there should be a function to print data (scores of all users and the average) taking as an input the results of the previous two functions